### ISSUE TITLE
Auto load next image

### ISSUE THREAD

**Issue by cyberb** [owncloud/android#932]
Auto loading of the next picture while viewing current one would improve usability.
Nice small progress bar of the next picture loading process would also indicate when to switch to the next one.

**Comment by nedimhx** [comment]
That would be very great ! But this is not for issue, post that to Pull request :)

**Comment by tobiasKaminsky** [comment]
@cyberb the problem is that it could waste a lot of of bandwith if the user does not want to see the next picture.
I am currently developing a reduced preview version. This one is sized to the device screen size and is ~30-50kb. So this image will be loaded very fast and can safe a lot of bandwith.

@nedimhx No no. Bug reports, feature wishes and so on belong to issues. Pull requests are only for committing code.

**Comment by cyberb** [comment]
@tobiasKaminsky Agree, reasonable preview scaling is always good, I guess it must be server side scaling then. At the same time "Auto load next picture" maybe a setting for people with cheap bandwidth :)

**Comment by tobiasKaminsky** [comment]
@cyberb of course it is server side.
And maybe, if there is an dev who wants to implement your idea, it will get integrated.

**Comment by davivel** [comment]
Let me rename the issue.

**Comment by DeepDiver1975** [comment]
🤖 Automated issue triage by Claude Code review agent.

This issue was opened before 2020 and has had no activity since then, so we are
closing it as outdated. If this is still relevant on a current ownCloud release,
please **reopen** (or open a fresh issue) with up-to-date details — thank you\!
